changelog.d/2967.breaking.rst

```diff
@@ -0,0 +1 @@
+The main bot config is no longer directly accessible to cogs. New methods have been added for use where this is concerned.
```

Comment:
Should probably explicitly add mentions to new functions in the changelog for those who need to account for the breaking changes, especially the API key stuff